### ISSUE TITLE
Not launching some features

### ISSUE THREAD

**Issue by gguzman86** [AequilibraE/qaequilibrae#25]
Hi Pedro,
first of all, I really value what you are doing with AequilibraE and is becoming a really good tool.

Regarding the issue at hand, there are problems launching "Shortest Path", "Impedance Matrix", "Traffic Assignment" and "Desired Lines". I got the message `Binary Error: Please download it from the repository using the downloader from the menu`. But I have the Scripts.

Also, I opened "impedance_matrix_procedures.py" and realized that you had your personal file path, so you may want to take a look at that.

`import qgis
from qgis.core import *
from PyQt4.QtCore import *
import time
import numpy as np
import sys
sys.path.append("C:/Users/Pedro/.qgis2/python/plugins/AequilibraE/")`

As a last thing, in "Parameters" you had your personal file path as well.

Regards,
Gustavo.

**Comment by pedrocamargo** [comment]
Thanks,  Gustavo.  I'll address it today.
Cheers,
Pedro

**Comment by pedrocamargo** [comment]
Gustavo,
                 I fixed the issue with the personal line of code, but this new version DOES require you to download the binaries afterwards. It is a requirement of the QGIS community. Let me know if things work and we can close the issue.

**Comment by gguzman86** [comment]
Hi Pedro,
I think the issue is that downloading the files from your website http://www.aequilibrae.com/ are not the same as downloading from the repository. I downloaded the ones from the repository and most of them work except Impedance Matrix. I solved this issue by putting the files impedance_matrix_dialogs.py and impedance_matrix_procedures.py in the folder "AequilibraE-master".

After that fix, this issue would be close.
